### DISCUSSION TITLE
Create git tag automatically on merge

### DISCUSSION THREAD

**Discussion by You-Honey** [renovatebot/renovate#11238] [Request Help] [answered]
When automatic merge is used is it possible to have Renovate create a tag too? This tag would trigger cloud build.

If it's not possible with Renovate can you give me a tip how I could do it with GitHub actions?

**Comment by rarkins** [comment, marked as answer]
It's not currently supported. If doing it with actions you would want to run it on commits to main/master, which is pretty easy, then check the commit author to see it's Renovate, and then add the tag either using the API or using git. API seems easier.
